test/helper/marketHelper.ts

```diff
@@ -34,6 +38,13 @@ export async function initMarket(
         await fixture.exchange.setMaxTickCrossedWithinBlock(baseToken, maxTickCrossedWithinBlock)
     }
 
+    if (isForwardTimestamp) {
+        const clearingHouse = fixture.clearingHouse as TestClearingHouse
+        await initiateBothTimestamps(clearingHouse)
```

Comment:
Two questions
1. Not sure why we add this logic here cuz I'm still seeing `initiateBothTimestamps` in such as `ClearingHouse.badDebt.test.ts`. Do we simply forget to remove it?
2. Since we `initiateBothTimestamps` already, I think we don't need to `forwardBothTimestamps` again, right?

Reply:
1. No. It's required.
In most test cases, we need to deploy market at the beginning of beforeEach  and setup some pre-work for each test case by sending couples of transactions.  Due to we need to calculate mark price, so we initial and forward both of bloc timestamps in `initMarket` function. But in some tests for funding payment, the values of mock block timestamp and evm block timestamp are gradually separated after some pre-work transactions (ex: adding liquidity), so it's better to initial both block timestamps again to make funding payment test easier and prevent flaky funding payment tests.

2. forwardBothTimestamps is for mark price calculation.

Reply:
Not getting your point haha
Will gather you when you're available!

Reply:
After discussed, will use forwardRealTimestamp to prevent `OLD` reverted by Uniswap contract while mark price calculation.
Fixed in be6bfb5d26faf4773183824cf927fe693557acf7